src/index.html

```diff
@@ -17,6 +17,85 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="logo__link">
+        <a href="/">
+          <img
+            src="images/logo.png"
+            alt="Company logo"
+          />
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="list__item">
+            <a
+              href="index.html"
+              class="nav__item is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="list__item">
+            <a
+              href="index.html"
+              class="nav__item"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="list__item">
+            <a
+              href="index.html"
+              class="nav__item"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="list__item">
+            <a
+              href="index.html"
+              class="nav__item"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="list__item">
+            <a
+              href="index.html"
+              class="nav__item"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="list__item">
+            <a
+              href="index.html"
+              class="nav__item"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="list__item">
+            <a
+              href="index.html"
+              class="nav__item"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="list__item">
+            <a
+              href="index.html"
+              class="nav__item"
+            >
+              Video
+            </a>
+          </li>
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. 'list__item' should be renamed to something like 'nav__item' to match its role within the 'nav__list'.